htlcswitch/switch.go

```diff
@@ -1038,7 +1038,9 @@ func (s *Switch) handlePacketForward(packet *htlcPacket) error {
 
 			return s.failAddPacket(packet, failure, addErr)
 		}
-		interfaceLinks, _ := s.getLinks(targetLink.Peer().PubKey())
+		targetPeer := targetLink.Peer()
+		targetNodeKey := targetPeer.IdentityKey().SerializeCompressed()
```

Comment:
Just call `targetPeer.PubKey()` directly?

Reply:
:man_facepalming: right, fixed and removed the extra var.